pkg/schemas/gen/main/main.go

```diff
@@ -8,6 +8,8 @@ import (
 	"slices"
 	"strings"
 
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/internal/genhelpers"
```

Comment:
why this import is added here?

Reply:
As I remember the build was failing because of it. It's used in the main function to call `NewGenerator`.